### PR TITLE
Addition for interoperability with other plugins

### PR DESCRIPTION
**Pull request by sjones6**
@PsychoLlama : Realized that this was missing. If it's not there, the options chain stops (no callbacks added via `Gun.on('opt')` will be called if added after this one.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -2,7 +2,11 @@
 import Adapter from './Adapter';
 import Gun from 'gun/gun';
 
-Gun.on('opt', (context) => {
+Gun.on('opt', function(context) {
+
+  // Pass to subsequent opt handlers
+  this.to.next(context);
```
Comment (PsychoLlama) [inline src/index.js]: Oooh, you're right, good catch.

File: src/index.js [hunk 2]
```diff
@@ -2,7 +2,11 @@
 import Adapter from './Adapter';
 import Gun from 'gun/gun';
 
-Gun.on('opt', (context) => {
+Gun.on('opt', function(context) {
```
Comment (PsychoLlama) [inline src/index.js]: The "Psycho" in "PsychoLlama" stands for my usage of spaces before a function paren. Changing that should fix the build

Comment (sjones6) [inline src/index.js]: haha! ;) the linter at work is just the opposite so now I'm in that habit. I fixed it up and the build passed.

**Comment by PsychoLlama** [comment]
Looks good to me 👍

**Comment by PsychoLlama** [comment]
@sjones6 heads up, I added you as a collaborator, so I'm pretty sure you can merge these once they pass. Stuff can be deployed to npm by adding a git tag and getting it on master (using `$ npm version patch && git push --tags` is easiest).

**Comment by sjones6** [comment]
@PsychoLlama : Awesome sauce. Thanks for taking care of the deploy on this go-round. 👍

**Comment by PsychoLlama** [comment]
Sure thing! Thanks for the contributions!

**Pull request merged**